tests/debugging/test_debugger.py

```diff
@@ -664,3 +665,25 @@ def count_status(queue):
 
         sleep(0.5)
         assert count_status(queue) == {"INSTALLED": 3, "RECEIVED": 2, "ERROR": 3}
+
+
+@pytest.mark.parametrize("duration", [1e5, 1e6, 1e7])
```

Comment:
Could you comment this to explain it a bit? I'm having trouble understanding how it's testing the duration.

Reply:
We are varying the duration of the wrapped function to ensure that the value that gets stored in the `duration` field is compatible with the actual function call duration. Here, `durationstuff` takes a duration as argument and will sleep for that amount of microseconds. We then check that we we snapshotted in the `duration` field is roughly the duration we passed to `durationstuff`.